icubam/db/store.py

```diff
@@ -756,8 +756,9 @@ def get_visible_bed_counts_for_user(
     kargs are passed to get_latest_bed_counts_for_icus() method for additional
     filtering, e.g. max_date.
     """
-    # TODO: handle user=None here (#180)
     user = self.get_user(user_id)
+    if not user:
+      raise ValueError(f"Cannot find user with id {user_id}")
```

Comment:
Please add "closes #180" to the issue description to auto-close.

Reply:
Right, done, thanks!